textworld/generator/inform7/world2inform7.py

```diff
@@ -32,6 +32,26 @@ class CouldNotCompileGameError(RuntimeError):
     pass
 
 
+def split_string(string, name, cutoff=200):
+    source = ""
```

Comment:
@tavianator 
I need to add a docstring and comment this function. Basically, it's there because Inform7 has a length limit for text inside quotation marks, i.e. `The description of the objective is "bla bla ... bla"`.

Also, this function needs to take some precaution to avoid breaking Inform7 tags, e.g. `This is a line [line break] break!`. At the moment, it only handles `[line break]` tags and I don't know how to make it robust to other tags as well as conditional tag, e.g. `[if chest open]It's open[otherwise]It's closed[end if]` should be split in any way!